tests/acceptance/testall

```diff
@@ -684,6 +684,13 @@ while true; do
   shift
 done
 
+#
+# Close stdin file descriptor to avoid tty_interactive check in cf-agent being success.
+#
+if ["$GDB" != 1] || ["$USE_VALGRIND" != 1]; then
+  exec <&-
+fi
```

Comment:
I think POSIX compliance demands spaces between the ``[``, ``]`` and the rest. This might fail on the exotic Unixes without it.